integration_tests/jenkins_jobs/02_genesis_setting.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+cd $HOME/friday/integration_tests && \
+pip3 install -r ./requirements.txt && \
+PATH=$PATH:$HOME/go/bin && \
+python3 multinode_test_setup.py
```

Comment:
End of line character is omitted. It is a trivial thing, but if you run editorconfig plugin, the formats of the files would be corrected automatically~!

Reply:
https://github.com/hdac-io/friday/pull/74/commits/d3efaffc15743d1b3afc98330f0501eae46dd424